### ISSUE TITLE
astropy.test plugin in test function

### ISSUE THREAD

**Issue by astrophysaxist** [larrybradley/lacosmic#7]
Hi, thanks for coding up LA Cosmic in python! I was just taking a look at using this package in a Python 3.8 notebook, and I stumbled upon an error when I went to run test().

```
ImportError while loading conftest '/home/---/.local/lib/python3.8/site-packages/lacosmic/conftest.py'.
/home/---/.local/lib/python3.8/site-packages/lacosmic/conftest.py:5: in <module>
    from astropy.tests.pytest_plugins import *
E   ModuleNotFoundError: No module named 'astropy.tests.pytest_plugins'
<ExitCode.USAGE_ERROR: 4>
```

From some googling, I think this is related to this issue (https://github.com/astropy/ccdproc/issues/660#issue-410599894) with the way astropy includes pytest plugins nowadays (https://github.com/astropy/ccdproc/issues/660#issuecomment-470706371). Seems like it could be any easy fix to include some code like the one in this fix (https://github.com/astropy/ccdproc/pull/650) in their ccdproc/conftest.py file.

I could fork and try submitting a pull request; let me know what you think.

Best,
Greg

**Comment by astrophysaxist** [comment]
Just forked the repository, and see that this is fixed in the unreleased version (not the released PyPI v0.1.1). I've installed lacosmic version 0.1.2.dev68+gf5c71c9 and now test() works as expected!

**Comment by larrybradley** [comment]
Thanks, @astrophysaxist.  Yes, I updated the infrastructure a while back.  I'll make a new release soon.

Also, in case you haven't seen it, there's a much faster cosmic-ray rejection package that you may want to check out:  https://github.com/astropy/astroscrappy.  It's not *exactly* the LAComic algorithm, but it's close enough for all practical purposes.
